tests/workbenches/conftest.py

```
@@ -56,19 +56,33 @@ def default_notebook(
-    minimal_image_path = (
-        f"{INTERNAL_IMAGE_REGISTRY_PATH}/{py_config['applications_namespace']}/{minimal_image}"
-        if internal_image_registry
-        else ":" + minimal_image.rsplit(":", maxsplit=1)[1]
-    )
+    # Determine which image to use
+    if custom_image:
+        # Custom image provided - use it directly (must include full registry path with tag)
+        if not custom_image or ":" not in custom_image:
+            raise ValueError(
+                f"custom_image must be a full image URL with tag (e.g., 'quay.io/org/image:tag'), got: '{custom_image}'"
+            )
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)